packages/frontend/src/assets/fonts/index.ts

```diff
@@ -0,0 +1,17 @@
+import localFont from 'next/font/local';
+import { Inter } from 'next/font/google';
+
+export const fontHeading = localFont({
+  src: './Adamina-Regular.ttf',
+  variable: '--font-heading'
+});
+
+export const fontSubheading = localFont({
+  src: './Ramabhadra-Regular.ttf',
+  variable: '--font-subheading'
+});
+
+export const inter = Inter({
```

Comment:
**issue (blocking):**  Please could you review the implementation of the font as the design used inter for the auth screen but the screenshots show otherwise